<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButton_Back">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Forward">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Url"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>47</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Go">
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWebView" name="Message_Webkit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>71</height>
      </size>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_Url</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_Go</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_Back</sender>
   <signal>clicked()</signal>
   <receiver>Message_Webkit</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_Forward</sender>
   <signal>clicked()</signal>
   <receiver>Message_Webkit</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
